drivers/input/keyboard/brain-kbd-i2c.c

```diff
@@ -36,14 +47,45 @@ struct bk_i2c_data {
 	struct keymap_def *km_symbol;
 	int kmlen;
 	int kmlen_symbol;
-
 	bool symbol;
 	u32 symbol_keycode;
+	bool near_close;
```

Comment:
単に "closing" で良いです。こうすると、if や三項演算子で意味が汲み取りやすくなります。

```suggestion
	bool closing;
```